### ISSUE TITLE
Istio 1.0.1 Tracker

### ISSUE THREAD

**Issue by rkpagadala** [istio/istio#7672]
1.0.0 is released and it is an LTS release. We will create a patch release 1.0.1 on August 20th

If you have any critical issues* or PRs that should be considered for 1.0.1, please add them to this issue. 

(*) Critical Issues Definition
Regression of a feature that is used by customers in prior releases.
Runtime crashes that severely regress stability of the system (i.e. SLO)

**Comment by andelhie** [comment]
Add nodeSelector in Helm Chart for Ingress,IngressGateway #6974

**Comment by sdake** [comment]
Not sure why I can't add to the list of blocked list, but if someone would be kind enough to add: https://github.com/istio/istio/issues/7782

**Comment by duderino** [comment]
@sdake I can't add it either.  I wonder if the issue has to be in the open state to add it as a dependency?

**Comment by linsun** [comment]
https://github.com/istio/istio/pull/7825

**Comment by jasmine-jaksic** [comment]
#7789

**Comment by sdake** [comment]
https://github.com/istio/istio/issues/7943

**Comment by rsnj** [comment]
#7675

**Comment by gargnupur** [comment]
#7944

**Comment by rkpagadala** [comment]
Everybody please don't forget to create your PR both in release-1.0 branch and master branch. There will not be a merge by the release team.

**Comment by vadimeisenbergibm** [comment]
https://github.com/istio/istio/pull/8015

**Comment by vadimeisenbergibm** [comment]
https://github.com/istio/proxy/pull/1924

**Comment by sdake** [comment]
@rkpagadala re https://github.com/istio/istio/issues/7782 fair enough - its a defect but helm is in beta, and does not meet the critical issues definition.

**Comment by lei-tang** [comment]
Issue #7943. The fix has been merged in #7945 on the master branch. The #7945 is cherrypicked by #8044 on the release 1.0 branch.

**Comment by sdake** [comment]
https://github.com/istio/istio/pull/8055

**Comment by sdake** [comment]
https://github.com/istio/istio/pull/8091 - cherrypick here -> (waiting on master merge)

**Comment by hklai** [comment]
Summarizing the current state:

#6974 -> Postponed to 1.1.
#7782 -> Fixed by #7784 (merged to master, but not release-1.0. Not critical enough.)
#7825 -> Merged to release-1.0
#7789 -> Merged to release-1.0
#7943 -> Fixed by #8044 (merged to release-1.0)
**#7675 -> Still open.** 
#7944 -> Merged to release-1.0
#8015 -> Merged to release-1.0
istio/proxy#1924 -> Merged to release-1.0
#8055 -> Merged to release-1.0
**#8091 -> Merged to master.** 

It looks like there are only two remaining open items.
#8091 - @sdake are you going to cherrypick the fix to release-1.0?

#7675 - @douglas-reid what is your assessment? Do we need a fix in 1.0.1?

And did I miss anything?

**Comment by hklai** [comment]
It looks like #8091 is already in release-1.0 (#8194).
As for #7675, it seems like a resource related issue and there isn't anything we need in 1.0.1.

So it looks like we are good to cut 1.0.1 now. Any objection?


/cc @jasmine-jaksic

**Comment by hklai** [comment]
1.0.1 is out. yay!
https://github.com/istio/istio/releases/tag/1.0.1

**Comment by sdake** [comment]
@hklai apologies on lag on a cherrypick on #8091.  I had originally thought cherrypicking would be the correct action for this repo, but have rethought that and believe direct merge for the helm preps make more sense.

Cheers
-steve

**Comment by craigbox** [comment]
Do we compile 'release notes' or changelogs?  It seems like it would be a lot of work to reverse-engineer one from the [commit list](https://github.com/istio/istio/compare/3a136c90ec5e308f236e0d7ebb5c4c5e405217f4...af73923d620cfa52e4433a0102d0bbb80500d278).

**Comment by hklai** [comment]
Yes @geeknoid is preparing that.

**Comment by linsun** [comment]
yes - https://github.com/istio/istio.github.io/pull/2429
